Makefile

```diff
@@ -212,3 +217,47 @@ set-build-info:
 	rm buildinfo.json.bak
 	echo "(re)generated buildinfo.json"
 	cat buildinfo.json
+
+
+# This uses JSONNET build tooling to rebuild all kube-prometheus manifest YAML
+# files based on the file `conbench-flavor.jsonnet` (i.e, including
+# conbench-specific customizations). It took me a longish while to get this
+# working (and to briefly understand the JSONNET build chain). This method here
+# is what we currently use for kube-prometheus customization. Note that the
+# coreos/jsonnet-ci container image used below comes with `jq` (one could
+# install this locally with e.g. sudo dnf install jsonnet) and also with
+# gojsontoyaml (which is where I resorted to looking for a container image that
+# has all dependencies baked in).
+.PHONY: jsonnet-kube-prom-manifests
+jsonnet-kube-prom-manifests:
+#	rm -rf _kpbuild
```

Comment:
Not removing this means much faster rebuilds (2 seconds instead of 1 min, roughly). I was on the edge. Will see what my local dev experience looks like.